upup/pkg/fi/cloudup/awstasks/launchtemplate_target_terraform_test.go

```diff
@@ -61,8 +62,9 @@ resource "aws_launch_template" "test" {
   instance_market_options {
     market_type = "spot"
     spot_options {
-      block_duration_minutes = 60
-      max_price              = "0.1"
+      block_duration_minutes         = 60
+      instance_interruption_behavior = "hibernate"
```

Comment:
something is strange with the whitespace here. the failing test shows:

```
          instance_market_options {
            market_type = "spot"
            spot_options {
              block_duration_minutes 	     = 60
        	  instance_interruption_behavior = "hibernate"
        	  max_price              		 = "0.1"
            }
          }
```

If you try selecting that text with your cursor you'll see various tab characters rather than individual spaces. This text "expected" string should use only spaces for indentation.

That said, when I check out your branch I only see space characters so I'm not sure whats wrong exactly :/ can you review this file, perhaps with a different text editor, and confirm it only uses spaces?